### ISSUE TITLE
Split-apk is not supported when cloning

### ISSUE THREAD

**Issue by akosiaris** [PeterCxy/Shelter#127]
Trying to clone youtube app to the work profile I get back the following

```
I Finsky  : [221] aboj.c(10): Package com.google.android.youtube blocked since it is missing required splits.
```

The device is a lineageos 17.1 (android 10), with shelter 1.5 from f-droid. The work profile is not associated with some gmail user and has been just created (and recreated a couple of times for ruling out some state messing with it). Other applications get cloned successfully however.

**Comment by PeterCxy** [comment]
I think I should create a central issue for these split apks.

**Comment by PeterCxy** [comment]
Or rather, I'll just edit this one instead

**Comment by PeterCxy** [comment]
For now, I would suggest getting some sort of app store working inside the work profile and install these apps from them.

**Comment by akosiaris** [comment]
> For now, I would suggest getting some sort of app store working inside the work profile and install these apps from them.

Yes, this tends to work and I 've used it already to sidestep the problem, but part of the beauty of this app is to avoid linking the apps to an app store.

Anyway, thanks for the effort put into this app, it's really nice.

**Comment by PeterCxy** [comment]
> but part of the beauty of this app is to avoid linking the apps to an app store.

Definitely. I will be working on this issue soon.

**Comment by PeterCxy** [comment]
Fixed in 2911a1d.
